src/Components/Web/src/Head/HeadOutlet.cs

```diff
@@ -14,8 +14,8 @@ public sealed class HeadOutlet : ComponentBase
 {
     private const string GetAndRemoveExistingTitle = "Blazor._internal.PageTitle.getAndRemoveExistingTitle";
 
-    internal const string HeadSectionOutletName = "head";
-    internal const string TitleSectionOutletName = "title";
+    internal const string HeadSectionOutletName = "blazor_head";
+    internal const string TitleSectionOutletName = "blazor_title";
```

Comment:
This is good 👍

I do wonder if we could go even further and replace the `Name` parameter on `SectionContent` and `SectionOutlet` with an `object` parameter, and call it `Id`? This would enable libraries to be completely certain they're not picking a `Name` that conflicts with another library (or the user's app).

You could do something like:

`MyLibraryComponent.razor`:
```razor
<h1>This is a component in an RCL<h1/>

@*The app can inject additional content here*@
<SectionOutlet Id="SectionOutletId">

@code {
    public static readonly object SectionOutletId = new();

    // ...
}
```

`MyAppComponent.razor`
```razor
<h1>This is a component in my app<h1/>

@*Inject some content into the <MyLibraryComponent /> somewhere else in the app*@
<SectionOutlet Id="MyLibraryComponent.SectionOutletId">
    <span>Here is some content</span>
</SectionOutlet>
```

This would also enable the scenario where there are potentially multiple instances of a component rendering a `<SectionOutlet />`. You could make the `SectionOutletId` non-static and let other components pick which "instance" to render to (via a cascading value, DI service, etc.).

Do you/others have any thoughts on this? @dotnet/aspnet-blazor-eng

Reply:
> At the same time I am fine with using string `Name` instead of object `Id` as it is a convention in Html to use string for elements ids.

Yeah, I can see how naming the component parameter `Id` could be seen as somehow relating to the `id` HTML element attribute. Perhaps using a different parameter name would avoid this problem, maybe `Identifier` or `SectionKey`?

Reply:
I think `SectionId` sounds good.